msg/en_US/csql.msg

```diff
@@ -163,7 +163,9 @@ $ csql help messages
    ;COMmit                      - commit the current transaction.\n\
    ;ROllback                    - roll back the current transaction.\n\
    ;AUtocommit [ON|OFF]         - enable/disable auto commit mode.\n\
-   ;REStart                     - restart database.\n\
+   ;REStart                     - reconnect to the target database in a CSQL session.\n\
+   ;CHeckpoint                  - execute the checkpoint.(CSQL with --sysadm only)\n\
+   ;Killtran                    - check or abort specific transaction.\n\
```

Comment:
;Killtran                    - check or abort specific transaction (CSQL with --sysadm only).\n\

Reply:
Right. I missed it.Thank you.